src/main/resources/static/src/views/SignUp/SignUp.js

```diff
@@ -40,13 +40,14 @@ export default class SignUp extends React.Component {
             .then(function () {
                 _this.setState({registrationIsSuccessful: true});
                 _this.setState({registrationIsNotSuccessful: false});
-                _this.autoLogIn(user.email, user.password)
+                _this.autoLogIn(user.email, user.password);
+                document.cookie = user.email;
             })
             .catch(function () {
                 _this.setState({registrationIsSuccessful: false});
                 _this.setState({registrationIsNotSuccessful: true});
             });
-
+        console.log("cookie: " + document.cookie);
```

Comment:
unused cookie declaration, and unsafe log also

Reply:
this part of code will be change when we add normal using cookies and i use this just for show my functional in my task!